src/api/java/mekanism/api/recipes/ChemicalOxidizerRecipe.java

```diff
@@ -0,0 +1,86 @@
+package mekanism.api.recipes;
+
+import java.util.List;
+import java.util.function.Predicate;
+import mekanism.api.MekanismAPI;
+import mekanism.api.annotations.NothingNullByDefault;
+import mekanism.api.chemical.merged.BoxedChemicalStack;
+import mekanism.api.recipes.ingredients.ItemStackIngredient;
+import net.minecraft.core.Holder;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.item.crafting.SingleRecipeInput;
+import net.minecraft.world.level.Level;
+import net.neoforged.neoforge.registries.DeferredHolder;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+/**
+ * Input: ItemStack
+ * <br>
+ * Output: Chemical
+ *
+ * @apiNote Chemical Oxidizers can process this recipe type.
+ */
+@NothingNullByDefault
+public abstract class ChemicalOxidizerRecipe extends MekanismRecipe<SingleRecipeInput> implements Predicate<@NotNull ItemStack> {
```

Comment:
shouldnt this extend `ItemStackToChemicalRecipe`?

Reply:
I don't know how, or if it's possible, to use ItemStackToChemicalRecipe without specifying one of the 4 chemical types that inherit from Chemical.  I'm open to suggestions!